apps/system/locales/system.en-US.properties

```diff
@@ -201,7 +201,7 @@ wantToInstallNow=Do you want to install the update now? You will not be able to
 installNow=Install Now
 notNow=Not Now
 # the <span> element will be displayed when the download finishes
-downloadingUpdateMessage.innerHTML=Downloading updates… <span>{{ progress }} downloaded.</span>
+downloadingUpdateMessage=Downloading updates… <span>{{ progress }} downloaded.</span>
```

Comment:
I don't know, but I don't think that this patch changes the behavior, right? If lack of `<span>` in the source HTML was a problem it would be visible without the patch as well, right?